funlet-call-me/.env

```diff
@@ -1,3 +1,9 @@
+# description: Calls made to your Twilio number will get forwarded to this E.164-formatted phone number
+# format: phone_number
+# required: true
+# link: https://www.twilio.com/docs/glossary/what-e164
+FUNLET_CALLME_PHONE_NUMBER=+12223334444
```

Comment:
is funlet a customer facing term?

Reply:
It's a portmanteau of "function" and "twimlet". I don't know how customer facing that makes this, but that's the origin.

Reply:
It might make sense to document Funlet as a customer-facing term, since it's something the customer will definitely encounter while working with these. It's also somewhat hard to rename environment variables for these while guaranteeing drop-in compatibility with the original Twimlets; I don't think `configure-env` has an option to give a variable a different display name than its variable name (though that'd be incredibly useful -- might be a good feature request).